controllers/MailController.php

```diff
@@ -309,7 +309,8 @@ public function actionCreate($userGuid = null)
         
         // Preselect user if userGuid is given
         if ($userGuid) {
-            $user = User::findOne(['guid' => $userGuid]);
+            /* @var User $user */
+            $user = User::find()->where(['guid' => $userGuid])->active()->visible()->filterBlockedUsers()->one();
```

Comment:
@yurabakhtin Requiring every module to use `->active()->visible()->filterBlockedUsers()` on user searches is a bit complicated. Maybe we find a simpler method here - especially for module usage?

Reply:
@luke- I can suggest to combine these 3 methods `->active()->visible()->filterBlockedUsers()` into one with name like `->searchable()`, do you agree?

Reply:
@yurabakhtin Hmm, I think we would use this filter in the MemberList widget, for example. In that context, searchable would not be so appropriate.

What do you think about `displayable()` or `readable()`.

Reply:
@luke- I think `displayable()` is more clear than `readable()`.
But it seems `displayable()` means almost same as `visible()`.
Maybe we also can use word like `available()`, because the method `filterBlockedUsers(?UserModel $user = null)` depends on current User's blocked users, so the `available()` will means a list of active/visible/displayable users which are available for current User.

Reply:
@yurabakhtin I like `available()` maybe we could also add the optional parameter `available(User $user)`, to determine for which user.

Reply:
@luke- Ok, core `develop` PR https://github.com/humhub/humhub/pull/5908, and commit here https://github.com/humhub/mail/pull/273/commits/0b8189619070fc3c2990b4a23f9500ad79f69aae.